### PR TITLE
Add email verification on Email signup

### LINKED ISSUE #1083: Add email verification on Email signup
This refers to "Issues in Staging" #1063

`Email signup accounts are not verified email identities/accounts.`

### PR DESCRIPTION
**Pull request by pdcp1**
Closes #1083

image

## Steps to test:

This is working here https://deploy-preview-5--pablocosta-staging.netlify.app/

Once you register a new user with email/password,  It will send you an email with the confirmation link.

This link is pointing to https://incidentdatabase.ai (ie: https://incidentdatabase.ai/confirmemail?token=2b8d249899e9091cbda231ebb277fd5a5c0f86ae3ccd8772dc6d6dc69de9004b2cc9f8b9e252235515a84a335738eecf9bd132a1e6073584554ae5a279d6e66f&tokenId=6323a8fa6ea1cfa6c41e6145 )

you have to copy the relative path and paste it to that deploy preview URL to test the full workflow.

### CODE REVIEW

**Comment by cesarvarela** [comment]
Is this working somewhere?

**Comment by pdcp1** [comment]
@cesarvarela This is working here https://deploy-preview-5--pablocosta-staging.netlify.app/

Once you register a new user with email/password,  It will send you an email with the confirmation link.

This link is pointing to https://incidentdatabase.ai (ie: https://incidentdatabase.ai/confirmemail?token=2b8d249899e9091cbda231ebb277fd5a5c0f86ae3ccd8772dc6d6dc69de9004b2cc9f8b9e252235515a84a335738eecf9bd132a1e6073584554ae5a279d6e66f&tokenId=6323a8fa6ea1cfa6c41e6145 )

you have to copy the relative path and paste it to that deploy preview URL to test the full workflow.

**Review by cesarvarela — COMMENTED** [review]
I'm getting this error when pushing to my env 🤔 

image

File: README.md [hunk 1]
```diff
@@ -526,6 +526,24 @@ Redirect URIs, is the URL that the user will be redirected to after successfully
...
+## Email subscriptions
+
+[Sendgrid](https://sendgrid.com/) is used to send email notifications.
+
+To allow email notifications add these secret values to your Atlas App Service following the instructions in the [Atlas App Services documentation](https://www.mongodb.com/docs/atlas/app-services/values-and-secrets/define-and-manage-secrets/).
+
+```
+SendGridApiKey = [SendGrid API key from https://app.sendgrid.com/settings/api_keys]
+publicApiKey = [Public API key from the Atlas Organization. See comment below for more information]
+publicApiKey = [Private API key from the Atlas Organization. See comment below for more information]
```
[the remark this answers is not in the document]
Comment (pdcp1) [reply README.md]: Fixed

**Comment by pdcp1** [comment]
@cesarvarela try again pushing to your env. It should be good now

**Review by cesarvarela — COMMENTED** [review]
I'm getting this error after changing from the production url to localhost:

image


image

**Review by cesarvarela — COMMENTED** [review]
The build has some warnings 🤔 
image

**Comment by pdcp1** [comment]
@cesarvarela the warning was fixed

**Pull request merged**